index.html

```diff
@@ -127,7 +127,7 @@ <h2>Main Program</h2>
             <div class="col-sm-6 col-md-2 col-lg-2 top-program-1">
               <div class="top-program">
                 <img src="./images/icons/exhibition.png" alt="exhibition-image">
-                <p>ABCD</p>
+                <p>Destination</p>
               </div>
               <div class="top-program-2">
                 <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia</p>
```

Comment:
Hello @richardoppiyo, you are doing a good job on this capstone project 💪🏻! 

- [ ] I would like to point out that the section below needs a bit more padding at the top and bottom as you can see in the behance design. 
- [ ] On further inspection main program cards are not well centered in the middle of the section as seen in the behance design.
- [ ] I am also suggesting that the cards are put into `<article>` tags as opposed to the `<div>`. This will help in making your code more readable and semantic.


| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/152214644-9983f27c-cb83-4502-a61a-725b57550238.png) | ![image](https://user-images.githubusercontent.com/4247740/152214418-7a7a54d7-4fb1-4db2-8d43-64317e131fcc.png) |

Reply:
Noted, the changes are going to be implemented with immediate effect.
Thank you for your feedback @Cathella